mantle/cmd/kola/testiso.go

```diff
@@ -335,6 +335,23 @@ RequiredBy=coreos-installer.target
 # for target system
 RequiredBy=multi-user.target`, nmConnectionId, nmConnectionFile)
 
+// Unit to check that if there is any failed service
+var verifyNoFailedService = `[Unit]
+Description=TestISO Verify No Failed Service
+OnFailure=emergency.target
+OnFailureJobMode=isolate
+After=rpm-ostree-fix-shadow-mode.service
+Before=live-signal-ok.service
+[Service]
+Type=oneshot
+RemainAfterExit=yes
+ExecStart=/bin/sh -c '/usr/bin/systemctl list-units --state failed --output json | grep -v failed'
```

Comment:
Will update to `ExecStart=/bin/sh -c '[[ -z $(/usr/bin/systemctl list-units --state failed --quiet) ]]'`